File: Iniciante/2779.py
espacos = int(input());
totalCompradas = int(input());
figurinhas = [];
contador = 0;

for i in range(totalCompradas):
    n = int(input());
    
    if (n not in figurinhas):
        figurinhas.append(n);
        contador += 1;

if (espacos >= contador):
    print(espacos-contador);
else:
    print(0);

File: Iniciante/2779.js
let espacos = Number(lines.shift());
let totalCompradas = Number(lines.shift());
let figurinhas = [], contador = 0;

for (let i = 0; i < totalCompradas; i++) {
  let n = Number(lines.shift());

  if (!figurinhas.includes(n)) {
    figurinhas.push(n);
    contador++;
  }
}

if (espacos >= contador) {
  console.log(espacos - contador);
} else {
  console.log(0);
}
